### PR TITLE
EIWFY23Q4-1 Webpack and storybook fixes

### PR DESCRIPTION
**Pull request by slatlasdev**
Nothing too interesting in this batch of changes, all maintenance stuff.

I noticed that Storybook had been broken by some of the previous changes (Monaco moving to ESM) so upgraded it to the latest version. I was also getting install errors with the favicons plugin compiling Sharp because it was too old for M1, so upgraded that as well.

This addresses your feedback comments about reformatting the inconsistent indents, as well as updating the README. More exciting UI/feature related changes to come soon.

### CODE REVIEW

File: .storybook/main.js
Comment (slatlasdev) [inline .storybook/main.js:1]: Migrated to TS, see below

File: src/Docs.tsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ export const Docs: React.FC = () => {
             const fileContents = await fetch(docsUrl).then(r => r.text());
             setLoadResult({ fileContents, loadId: id });
          } catch (e) {
-            setLoadResult({ message: e.message, loadId: id });
+            setLoadResult({ message: e instanceof Error ? e.message : JSON.stringify(e), loadId: id });
```
Comment (slatlasdev) [inline src/Docs.tsx:55]: TSv5 has more strict catch clause checks (can't assume everything is an Error)

File: src/SchemaEditor.tsx
Comment (slatlasdev) [inline src/SchemaEditor.tsx:1]: This whole file is just indentation changes (3 spaces to 2)

File: src/stories/Button.stories.tsx
Comment (slatlasdev) [inline src/stories/Button.stories.tsx:1]: All these were sample stories included with Storybook

File: src/stories/assets/code-brackets.svg
Comment (slatlasdev) [inline src/stories/assets/code-brackets.svg:1]: More sample assets...

File: src/Docs.tsx (hunk 1 above)
Comment (robertmassaioli) [reply src/Docs.tsx:55]: Great! This is a big positive. :)

File: src/stories/Button.stories.tsx
Comment (robertmassaioli) [reply src/stories/Button.stories.tsx:1]: Don't know why I did not delete them. Interesting.

**Review by robertmassaioli — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -33,42 +33,38 @@
...
     "css-loader": "^5.0.1",
-    "favicons": "^6.2.0",
-    "favicons-webpack-plugin": "^5.0.0-alpha.6",
-    "html-webpack-plugin": "^5.0.0-beta.4",
-    "http-server-spa": "^1.3.0",
+    "favicons": "^7.1.3",
+    "favicons-webpack-plugin": "^6.0.0",
+    "html-webpack-plugin": "^5.5.3",
     "json-schema-to-typescript": "^10.1.2",
-    "npm-run-all": "^4.1.5",
```
Comment (robertmassaioli) [inline package.json:63]: @slatlasdev I was using that for the package.json scripts. Can you add it back please?

Comment (slatlasdev) [reply package.json:63]: Sure thing. 
Although if there's common sets of scripts you run together, it's often a good idea to combine them in the `package.json` for maintainability and to share with others, eg:
```json
"deploy": "npm run build-prod && npm run deploy"
```

**Pull request merged**